src/app/components/nav/Nav.js

```diff
@@ -25,13 +26,12 @@ function NavigationBar() {
 						<NavLinkItem end to="/">
 							Home
 						</NavLinkItem>
-						<NavLinkItem to="/starter-packs">Resources</NavLinkItem>
-						<NavLinkItem to="/schedule">Schedule</NavLinkItem>
-						{/*<NavLinkItem to={{ pathname: APPLICATION_URL }}*/}
-						{/*             target="_blank"*/}
-						{/*             className="nav-link-outline">*/}
-						{/*    Apply*/}
-						{/*</NavLinkItem>*/}
+						{/* <NavLinkItem to="/starter-packs">Resources</NavLinkItem> */}
+						{/* <NavLinkItem to="/schedule">Schedule</NavLinkItem> */}
+						<NavLinkItem as="a" href={APPLICATION_URL} target="_blank" rel="noreferrer" style={{ color: "white"}}>
```

Comment:
https://stackoverflow.com/questions/61444735/custom-css-doesnt-override-navlink-in-react-bootstrap

Reply:
got it, we can look into a cleaner solution later